src/client.ts

```diff
@@ -354,6 +375,10 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
         case 'double':
           variableObj[key] = this._client.getFeatureVariableDouble(featureKey, key, userId, userAttributes);
           break;
+
+        case 'json':
+          variableObj[key] = this._client.getFeatureVariableJson(featureKey, key, userId, userAttributes);
```

Comment:
pleae change this reference to `getFeatureVariableJSON`. This has been updated in javascript SDK